### PR TITLE
Add bank to Candor

### PR DESCRIPTION
**Pull request by Cassy1**
Title says it all :-)
Goes with https://github.com/themanaworld/tmwa-client-data/pull/236

### CODE REVIEW

**Comment by Cassy1** [comment]
wtf is world/map/npc/018-2/.directory  ?
How do get rid of it?

**Comment by mekolat** [comment]
@Cassy1 it got added because you are using Dolphin File Manager
you can remove it by using "git rm"

File: world/map/npc/018-2/.directory [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[Dolphin]
+Timestamp=2015,2,15,18,31,35
+Version=3
```
Comment (wushin) [inline world/map/npc/018-2/.directory]: Cassy?

File: world/map/npc/029-2/bankroom.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Nyle is part of the Tutorial.
+// He explains how to use the bank.
+// Author: Cassy
+
+029-2,36,122,0|script|#BankRoom|45,1,0
+{
+message strcharinfo(0), "The door is locked. This must be where Cynric saves the people's money and items.";
```
Comment (wushin) [inline world/map/npc/029-2/bankroom.txt]: Indent

**Comment by Cassy1** [comment]
Oki, thanks, I will care about later when I'm home.

File: world/map/npc/029-2/nyle.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "And how do I save money?", L_Money,
+        "Thanks!", L_Thanks;
+
+L_Money:
+    mes "[Nyle]";
+    mes "\"That's easy. Cynric will save your money when you ask him to deposit. Or you pick up your money by asking to withdraw.";
+    mes "You only have to tell him how much you want to deposit or withdraw.";
+    mes "Also you can ask how much he already save for you by checking your balance.";
+    next;
+    mes "Money in your storage are totally save. Banking itself is totally save.\"";
```
Comment (Cassy1) [inline world/map/npc/029-2/nyle.txt]: Note to self: "Money in your storage *is* totally save"

**Comment by Cassy1** [comment]
"This pull request contains merge conflicts that must be resolved."
Do I need to rebase, then force-push again?

**Comment by wushin** [comment]
Yes

**Comment by wushin** [comment]
@Cassy1 Pls update?

**Pull request closed without merging**